c2corg_ui/static/menu-discourse/menu.html

```diff
@@ -0,0 +1,117 @@
+## HEAD
+<link href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap-glyphicons.css" rel="stylesheet">
```

Comment:
I have no admin rights :P 

So you mean both HTML and CSS codes have to be pasted in a form in the admin interface, right?